tests/foreman/api/test_syncplan.py

```diff
@@ -1,3 +1,4 @@
+# pylint: disable=invalid-name
```

Comment:
yeah the problem is that `test_` is a mandatory word.  So all test would have a mandatory 5 letters text (`test_`) which leaves around 26 characters to fit everything else.  But yeah this is doable.